tests/main/views/test_search.py

```diff
@@ -14,11 +14,6 @@ def find_pagination_links(res_data):
         re.MULTILINE)
 
 
-def find_search_summary(res_data):
```

Comment:
Oh, you just moved an existing regex thing that's over two years old?  Probably just leave it as-is for now then.

Reply:
Yep.